### ISSUE TITLE
ClockifyClient.FindAllTimeEntriesForProjectAsync "hydrated" parameter makes no sense

### ISSUE THREAD

**Issue by topennanen** [Morasiu/Clockify.Net#62]
As the method can only return unhydrated time entries (TimeEntryDtoImpl) , the parameter "hydrated" is useless.

Suggestion: Separate method for hydrated time entries, which uses appropriate method for the use time entry queries (ClockifyClient.FindAllHydratedTimeEntriesForUserAsync) and returns HydratedTimeEntryDtoImpl

**Comment by Morasiu** [comment]
I will check it with current API. Thanks for suggestion!

**Comment by Morasiu** [comment]
I see I already have `FindAllHydratedTimeEntriesForUserAsync`. Is it enough for your needs?

**Comment by topennanen** [comment]
Not really, no. The use case is finding all time entries for a task, regardless of the user, along with the names of the users who made those entries.

**Comment by topennanen** [comment]
That, or `FindAllHydratedTimeEntriesForProjectAsync` for a wider usage in line with the current non-hydrated method.

**Comment by Morasiu** [comment]
Roger that, will do :)

**Comment by Psypher9** [comment]
Looks like there's no need for this right now
